frontend/app/components/Assist/components/AssistActions/AssistActions.tsx

```diff
@@ -50,7 +43,8 @@ function AssistActions({
   agentIds,
   userDisplayName,
 }: Props) {
-  const { player, store } = React.useContext(PlayerContext)
+  // @ts-ignore ???
+  const { player, store } = React.useContext<ILivePlayerContext>(PlayerContext)
```

Comment:
TS is probably swearing here because of the type narrowing.
PlayerContext is parameterised by ContextType which is union of live and no-live.

Reply:
Probably yes, no good solution here without splitting player context into 2 parts